docs/it/università-vs-self-made.md

```diff
@@ -0,0 +1,44 @@
+# Università VS Self-Made
+
+## La terza via: l’ITS
+
+Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è, a tutti gli effetti, un bivio: _intraprendere la via dell’Università o iniziare un percorso di autoformazione?_
+
+La risposta non è sempre così scontata, vista anche la soggettività del tema. Nel primo caso stiamo parlando di un percorso che dura minimo tre anni (o cinque, a seconda dell’obiettivo), strutturato e tenuto da docenti formati e che seguono un piano di studi ben preciso, mentre nel secondo abbiamo una durata potenzialmente ignota e che si basa esclusivamente sulla nostra organizzazione, disciplina e motivazione.
+
+Molte persone però non sono a conoscenza di una terza via, ovvero degli [ITS (_Istituto Tecnico Superiore_)](https://www.miur.gov.it/web/guest/tematica-its) di programmazione, degli enti solitamente finanziati dalla regione e/o da alcune aziende del settore, il cui scopo è quello di formare nel giro di un paio d’anni delle figure professionali da inserire nel mondo dell’IT.
+
+### Un ibrido tra Teoria e Pratica
+
+**_“Com'è strutturato il percorso formativo di un ITS?”_**
+
+Per integrare in modo corretto e graduale la “Teoria” e la “Pratica”, il piano formativo standard di un ITS prevede:
+
+- **Primo Anno**: Prevalentemente dedicato alle **lezioni in aula**, dove si ha l'opportunità di apprendere dalle competenze di **docenti aziendali**<sup>1</sup> e universitari, i concetti, i linguaggi e gli strumenti fondamentali della figura professionale a cui è dedicato il corso.
+  Inoltre c’è da menzionare che in alcuni ITS viene offerta la possibilità di prepararsi e sostenere esami per certificazioni esterne quali: [CISCO IT Essentials](https://www.netacad.com/courses/os-it/it-essentials), [ECDL](https://www.aicanet.it/aica/qualita/ecdl-full-standard) e [PET/FIRST/ecc.](https://www.cambridgeenglish.org/exams-and-tests/) per citarne solo alcune.
+
+- **Secondo Anno**: Dopo una breve fase di lezioni dedicate agli ultimi moduli, mirate a esplorare concetti più avanzati, come l'_Architettura del Software_ per esempio, l'attenzione si sposta principalmente sullo **stage lavorativo**, nonché ultima attività prima dell’esame finale (necessario per conseguire un _Diploma di [V livello EQF](https://europa.eu/europass/it/description-eight-eqf-levels)_).
```

Comment:
Quali sono questi moduli? Trovo questo paragrafo poco chiaro

Reply:
Ciao, non sono entrato troppo nel merito in quanto questi moduli possono variare da corso a corso.
Ho citato l'architettura software siccome è stato uno di questi moduli "avanzati" che ho affrontato personalmente nel mio corso; ma come detto, è un esempio.
Se sei d'accordo posso cercare di rendere il tutto un po' più chiaro citando questo aspetto sulla differenza da corso a corso

Reply:
Secondo me non è sbagliato dire "Io ho fatto questo e questi sono i moduli - che ogni ITS struttura a modo suo"

Reply:
L'aggiunta della parola "generalmente" in alto per me è sufficiente per indicare che quello che si mostra è un esempio e non l'unica via. Che ne dici @AngeloAvv ?

Reply:
Va bene!